src/main/resources/application-local.yaml

```diff
@@ -0,0 +1,2 @@
+logging.level.root: debug
+logging.level.org.springframework.web: debug
```

Comment:
Do we want logging level as `debug` on master once this gets merged?

Reply:
this should not create any difference in the AAT envs or the PROD envs, since the "local" profile is left for local development only. And when developing ti's useful and quick to turn on the debug mode.